multi/grlib-multi/uart.c

```diff
@@ -49,6 +60,13 @@
 #define UART_SCALER 3
 #define UART_DEBUG  4
 
+/* Byte offset of LSB of UART_DATA register */
+#if defined(__BIG_ENDIAN)
```

Comment:
`__BIG_ENDIAN` is always defined
```suggestion
#if __BYTE_ORDER == __BIG_ENDIAN
```